python/python/glide/async_commands/server_modules/ft.py

```diff
@@ -181,3 +181,52 @@ async def aliasupdate(
     """
     args: List[TEncodable] = [CommandNames.FT_ALIASUPDATE, alias, indexName]
     return cast(TOK, await client.custom_command(args))
+
+
+async def info(client: TGlideClient, indexName: TEncodable) -> Mapping[
+    TEncodable,
+    Union[
+        TEncodable,
+        int,
+        List[TEncodable],
+        List[
+            Mapping[
+                TEncodable,
+                Union[TEncodable, Mapping[TEncodable, Union[TEncodable, int]]],
+            ]
+        ],
+    ],
+]:
+    """
+    Returns information about a given index.
+
+    Args:
+        client (TGlideClient): The client to execute the command.
+        indexName (TEncodable): The index name for which the information has to be returned.
+
+    Returns:
+        Mapping[TEncodable, Union[TEncodable, int, List[TEncodable], List[Mapping[TEncodable, Union[TEncodable, Mapping[TEncodable, Union[TEncodable, int]]]]]]]: Nested maps with info about the index. See example for more details.
+
+    Examples:
+        >>> from glide.async_commands.server_modules import ft
```

Comment:
from glide import ft

Reply:
Clarification: The actual file is still in server_modules. You are just saying to update in the documentation example?

Reply:
Updated.